wallet/txfilter.c

```diff
@@ -108,14 +108,6 @@ bool outpointfilter_matches(struct outpointfilter *of, const struct bitcoin_txid
 	return outpointset_get(of->set, &op) != NULL;
 }
 
-void outpointfilter_remove(struct outpointfilter *of, const struct bitcoin_txid *txid, const u32 outnum)
-{
-	struct outpointfilter_entry op;
-	op.txid = *txid;
-	op.outnum = outnum;
-	outpointset_del(of->set, &op);
-}
-
```

Comment:
@cdecker I assume this *should* be called from somewhere?

Reply:
I wanted to call this directly when marking them as spent, but then had to revert since reorgs would un-spend them again and then we need to track them again. I'll hook it up in the pruning after being buried deep enough. Feel free to remove for now, I'll re-add it when appropriate :-)

Reply:
Since this was a rather quick fix I opened PR #1279 addressing this. So the function is no longer unused :-p